addons/hr_fleet/models/employee.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import base64
+import io
+
+from odoo import api, fields, models
+
+class Employee(models.Model):
+    _inherit = 'hr.employee'
+
+    employee_cars_count = fields.Integer(compute="_compute_employee_cars_count", string="Cars")
+
+
+    def action_open_employee_cars(self):
+        self.ensure_one()
+        Fleet = self.env['fleet.vehicle.assignation.log']
+        car_line = Fleet.search([('driver_id','=',self.user_id.partner_id.id)]).mapped('vehicle_id')
+
+        return {
+            "type": "ir.actions.act_window",
+            "res_model": "fleet.vehicle",
+            "views": [[False, "kanban"], [False, "form"], [False, "tree"]],
+            "domain": [("id", "in", car_line.ids)],
+            "name": "History Employee Cars",
+        }
+
+    def _compute_employee_cars_count(self):
+        Fleet = self.env['fleet.vehicle.assignation.log'].sudo()
```

Comment:
Why is there a sudo here ? Should be removed IMO. If you have a traceback on the form view, it would be better to add the group fleet_manager on the field.